Makefile.checker

```diff
@@ -77,9 +77,8 @@ checker/%.cmx: checker/%.ml
 
 md5chk:
 	$(SHOW)'MD5SUM cic.mli'
-	$(HIDE)v=`tr -d "\r" < checker/cic.mli | $(MD5SUM) | sed -n -e 's/ .*//' -e '/^/p'`; \
-	       if grep -q "$$v" checker/values.ml; \
-	       then true; else echo "Error: outdated checker/values.ml: $$v" >&2; false; fi
+	$(HIDE)if grep -q "`$(OCAML) tools/md5sum.ml checker/cic.mli`" checker/values.ml; \
```

Comment:
The existing code handles the case on Windows where the git checkout has included return characters.  It's fine to use "$(OCAML) tools/md5sum.ml" but please do something like this to avoid breaking that:

```
$(HIDE)v=`tr -d "\r" < checker/cic.mli | $(OCAML) tools/md5sum.ml | sed -n -e 's/ .*//' -e '/^/p'`; \
 	       if grep -q "$$v" checker/values.ml; \
 	       then true; else echo "Error: outdated checker/values.ml: $$v" >&2; false; fi
```

Reply:
Ooh, ok I see. Sorry for that, I'll fixe it later tonight